packages/scss/src/components/inlineMessage/component.scss

```diff
@@ -6,7 +8,7 @@
   color: var(--components-inlineMessage-color);
 
   .lucca-icon {
-    font-size: var(--components-inlineMessage-icon-fontSize);
+    @include icon.XS;
     color: var(--components-inlineMessage-icon-color);
     margin-top: 2px;
```

Comment:
do we need to keep this margin instead of use `display flex` and `align-items center` on its parent ?

Reply:
I checked with Clément and our icon must stay on top if the message is displayed on multiple line, so we can't center it :'(